acceptance/tests/facts/macosx.rb

```diff
@@ -7,10 +7,10 @@
 # Facts tested: os, processors, networking, identity, kernel
 #
 
-confine :to, :platform => /osx-mavericks|osx-yosemite/
+confine :to, :platform => /osx-10\.9|osx-\.10/
```

Comment:
Is the second one missing a `10`? I'd check on the jenkins run but jenkins isn't responding for some reason ....